modules/tinymce/changelog.txt

```diff
@@ -1,4 +1,5 @@
 Version 5.3.1 (TBD)
+    Added names for menu items used by the table plugin. #TINY-4866
```

Comment:
`names` isn't really accurate here, it's more that you've added them to the registry so that they can be used. So maybe something like this instead?

> Added all table menu items to the UI registry, so they can be used by name in other menus #TINY-4866 

Also minor note, we don't normally have a fullstop before the JIRA reference.

Reply:
I also just noticed you added this to the wrong location in the changelog. This will be going into 5.4.0, not 5.3.1.

Reply:
I do have to ask, how do I know which version it'll get into? For the future.